src/Rest/RestApiBrowser.php

```diff
@@ -2,9 +2,13 @@
 
 namespace Rezzza\RestApiBehatExtension\Rest;
 
-use Guzzle\Http\Exception\BadResponseException;
-use Guzzle\Http\ClientInterface as HttpClient;
-use Guzzle\Http\Message\Response;
+use Ivory\HttpAdapter\HttpAdapterFactory;
+use Ivory\HttpAdapter\HttpAdapterInterface as HttpClient;
+use Ivory\HttpAdapter\HttpAdapterException;
+use Ivory\HttpAdapter\Message\Request;
+use Zend\Diactoros\Stream;
```

Comment:
We need to instantiate a real implementation in fact. So we need the `Zend\Diactoros\Stream`